docker-compose.yml

```diff
@@ -2,47 +2,56 @@ version: '3'
 
 services:
 
-  datastore:
-    image: knarz/datastore-emulator
-    networks:
-      - eq-env
-
-  redis:
-    image: redis:4
-    networks:
-      - eq-env
-
-  eq-questionnaire-runner:
-    image: onsdigital/eq-questionnaire-runner:latest
-    build: ./
-    env_file:
-      - ${RUNNER_ENV_FILE}
-    environment:
-      DATASTORE_EMULATOR_HOST: datastore:8432
-      EQ_REDIS_HOST: "redis"
-    restart: always
-    depends_on:
-      - datastore
-      - redis
-    networks:
-      - eq-env
-    ports:
-      - "5000:5000"
-
-  eq-questionnaire-launcher:
-    image: onsdigital/eq-questionnaire-launcher:latest
-    environment:
-      SURVEY_RUNNER_URL: http://localhost:5000
-      SURVEY_RUNNER_SCHEMA_URL: http://eq-questionnaire-runner:5000
-    restart: always
-    depends_on:
-      - eq-questionnaire-runner
-    networks:
-      - eq-env
-    ports:
-      - "8000:8000"
+    datastore:
+        image: knarz/datastore-emulator
+        networks:
+            - eq-env
+
+    redis:
+        image: redis:4
+        networks:
+            - eq-env
+
+    eq-questionnaire-runner:
+        image: onsdigital/eq-questionnaire-runner:latest
+        build: ./
+        env_file:
+            - ${RUNNER_ENV_FILE}
+        environment:
+            DATASTORE_EMULATOR_HOST: datastore:8432
+            EQ_REDIS_HOST: "redis"
```

Comment:
https://github.com/ONSdigital/eq-questionnaire-runner/pull/1187#discussion_r1297461180 needs to be done here as well.

Reply:
Thanks 👍 added it here